workspaces/orchestrator/README.md

```diff
@@ -171,115 +109,105 @@ yarn dev
+The user interface is accessible via the orchestrator button added in the Backstage sidebar. It provides a list of workflows and the option to run the workflows and view the results.
 
-### orchestrator-backend
+![user interface](./docs/userInterface.png)
 
-The orchestrator backend can use the generated schema to validate the HTTP requests and responses.
+## Permissions
 
-### audit log
+The Orchestrator plugin protects its backend endpoints with the builtin permission mechanism and combines it with
+the RBAC plugin. The result is control over what users can see or execute. Details available [here](./docs/Permissions.md).
 
-The orchestrator backend has audit logs for all incoming requests.
+## Orchestrator API
 
-For more information about audit logs in RHDH, please refer to [the official documentation](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/assembly-audit-log#con-audit-log-config_assembly-audit-log).
-[The official Log storage OpenShift documentation](https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/about-log-storage.html) may also be of interest.
+The backend plugin provides OpenAPI `v2` endpoints definition.
 
-#### Development instruction
+The OpenAPI specification file is availabe in the [openapi spec file](./plugins/orchestrator-common/src/openapi/openapi.yaml).
 
-Checkout the backstage-plugin
+Documentation is available in the autogenerated [documentation](./plugins/orchestrator-common/src/generated/docs/markdown/README.md)
 
-`git clone git@github.com:red-hat-developer-hub/backstage-plugins.git`
+The plugin provides an auto generated typescript client that can be used to call the API. To use it include the @red-hat-developer-hub/backstage-plugin-orchestrator-common plugin in your project. You can see in the [OrchestratorClient.ts](./plugins/orchestrator/src/api/OrchestratorClient.ts#L59) how the client can be used.
 
-If you need to change the OpenAPI spec, edit the [openapi.yaml](https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/orchestrator/plugins/orchestrator-common/src/openapi/openapi.yaml) according to your needs and then execute from the project root folder:
+## Audit log
 
-`yarn --cwd plugins/orchestrator-common openapi`
+The orchestrator backend has audit logs for all incoming requests.
 
-This command updates the [generated files](https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/orchestrator/plugins/orchestrator-common/src/generated) including API, client and docs.
+For more information about audit logs in RHDH, please refer to [the official documentation](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/assembly-audit-log#con-audit-log-config_assembly-audit-log).
```

Comment:
This can be updated to RHDH 1.4 doc:
https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/audit_log/assembly-audit-log